lib/task-data/base-tasks/ucs-service-profile-discovery.js

```diff
@@ -0,0 +1,17 @@
+// Copyright 2016, EMC, Inc.
+
+'use strict';
+
+module.exports = {
+    friendlyName: 'UCS Service Profile Discovery Job',
+    injectableName: 'Task.Base.UCS.Service.Profile.Discovery',
+    runJob: 'Job.Ucs.Service.Profile.Discovery',
+    requiredOptions: [
+        'uri'
+    ],
+    requiredProperties: {
+    },
+    properties: {
+        catalog: {}
+    }
+};
```

Comment:
@yyscamper if ok with the plan, can you please help remove the -1 that currently blocks the PR from being merged?